app/code/Magento/UrlRewrite/Test/Mftf/MetaData/url_rewrite-meta.xml

```diff
@@ -1,18 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!--
-/**
- * Copyright © Magento, Inc. All rights reserved.
- * See COPYING.txt for license details.
- */
--->
-
-<operations xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xsi:noNamespaceSchemaLocation="urn:magento:mftf:DataGenerator/etc/dataOperation.xsd">
-    <operation name="CreateUrlRewrite" dataType="urlRewrite" type="create" auth="adminFormKey" url="/admin/url_rewrite/save/" method="POST" successRegex="/messages-message-success/">
```

Comment:
You can not delete this operation cause here is data entities relying on it https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/UrlRewrite/Test/Mftf/Data/UrlRewriteData.xml#L11
And here is `createData` action which will use this data entity + operation on test runtime.
https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/UrlRewrite/Test/Mftf/Test/AdminUpdateCustomURLRewritesPermanentTest.xml#L22

Reply:
<img width="180" alt="Screenshot 2019-04-04 at 20 02 21" src="https://user-images.githubusercontent.com/1639941/55577782-a58db200-5714-11e9-95a7-2f3be0c65aee.png">

Once again you missed the case @okolesnyk

Reply:
🤦‍♂️My bad. You are right